drivers/android/binder.c

```diff
@@ -1432,6 +1432,26 @@ static void binder_send_failed_reply(struct binder_transaction *t,
 	}
 }
 
+/**
+ * binder_cleanup_transaction() - cleans up undelivered transaction
+ * @t:		transaction that needs to be cleaned up
+ * @reason:	reason the transaction wasn't delivered
+ * @error_code:	error to return to caller (if synchronous call)
+ */
+static void binder_cleanup_transaction(struct binder_transaction *t,
```

Comment:
This causes an unused variable warning because it's not used like in the original patch: f743ad82a49afae201ae8b24530c5832d979af4e

I guess you guys switched over to the Pixel 2's Oreo binder stack even though Google is moving back to kernel/common in P: https://github.com/aosp-mirror/kernel_msm/commit/29d196199717c5b49a236f2f0ced0f25bb0d03ed